docs/getting-started/tutorial-sample-discover.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[tutorial-sample-discover]]
+=== Using Discover
+
+Another way to interactively explore your data is in the Discover application.   
+Here the data is presented as a table of documents.  As in the Dashboard application, 
+you can submit search queries and filter the search results. 
+
+. In the side navigation, select *Discover*.
++
+The current index pattern appears below the filter bar. An
+index patterns tell {kib} which {es} indices you want to explore.
+You’ll learn more about index patterns in the next tutorial, 
+<<tutorial-your-data>>.
+
+. If *kibana_sample_data_flights* isn't the current index pattern, go to the menu
```

Comment:
how necessary do we think this is? If they're really getting started with Kibana there is a good chance they aren't seeing a saved search and need to remove it. And even if they have, they would have created it most likely. I don't think you can set a default saved search globally. It's possible they may have to select another index pattern but I don't think they'll need to hit `New`